std/node/process_test.ts

```diff
@@ -38,6 +40,7 @@ Deno.test({
     assert(process.cwd().match(/\Wnode$/));
     process.chdir("..");
     assert(process.cwd().match(/\Wstd$/));
+    process.chdir(currentDir); // to unchange current directory after this test
```

Comment:
Not strictly necessary is it? Node tests can be run from any directory as-is.

Reply:
@caspervonb yes node tests can be run with no changes but if I don't change this, some tests in `std/path/zero_length_strings_test.ts` fail
Because of `const pwd = Deno.cwd();`. this uses `Deno.cwd()` but current dir has been changed in tests in node

Reply:
Ah right, forgot we don't reset between test modules 😉

Reply:
`std/path/zero_length_strings_test.ts` could be rewritten to call `Deno.cwd()` in each test case

CC @nayeemrmn

Reply:
All tests should reset their state changes, so this change is fine and `zero_length_strings_test.ts` is fine as it is as well.

Tangential: I was wondering why cwd is used in `zero_length_strings_test.ts`... `path.resolve()` touches the filesystem and bases from there by default. I kind of hate that. Maybe it should just accept two paths or we can move it to `fs`.

Reply:
> Tangential: I was wondering why cwd is used in zero_length_strings_test.ts... path.resolve() touches the filesystem and bases from there by default. I kind of hate that. Maybe it should just accept two paths or we can move it to fs.

SGTM